runtime/parachains/src/system_token_manager.rs

```diff
@@ -56,6 +56,8 @@ pub type StringLimitOf<T> = <T as Config>::StringLimit;
 
 const REF_WEIGHT: u64 = 500_000_000;
 const PROOF_WEIGHT: u64 = 20_000;
+/// The base_system_token_weight(USD). Assume that it SHOULD not be changed.
```

Comment:
I think it would be better to remove keyword 'USD'

Reply:
Addressed it!